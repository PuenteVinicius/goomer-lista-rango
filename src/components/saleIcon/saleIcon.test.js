import React from 'react';
import SaleIcon from './saleIcon';
import { mount } from 'enzyme';
import toJson from "enzyme-to-json"

it("should render the homepage for the non logged in user", () => {
  const sale = {
    description: "Ter√ßa-feira doce",
    price: 5,
    hours: [
      {
        from: "09:00",
        to: "18:00",
        days:[3]
      }
    ]
  }
  const app = mount(<SaleIcon value={{ sale }}/>)
  expect(toJson(app)).toMatchSnapshot()
})
