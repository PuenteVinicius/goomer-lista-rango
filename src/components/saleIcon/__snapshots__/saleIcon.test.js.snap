// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the homepage for the non logged in user 1`] = `
<_default
  value={
    Object {
      "sale": Object {
        "description": "TerÃ§a-feira doce",
        "hours": Array [
          Object {
            "days": Array [
              3,
            ],
            "from": "09:00",
            "to": "18:00",
          },
        ],
        "price": 5,
      },
    }
  }
>
  <div
    className="sale"
  >
    <p
      className="sale__text"
    >
      <i
        className="fas fa-award sale__icon"
      />
    </p>
  </div>
</_default>
`;
